charts/jenkins/Chart.yaml

```diff
@@ -12,6 +12,8 @@ sources:
 - https://github.com/maorfr/kube-tasks
 - https://github.com/jenkinsci/configuration-as-code-plugin
 maintainers:
+- name: johny
+  email: shun.johny@gmail.com
```

Comment:
```suggestion
```
It would make sense if you contribute some more PRs or help reviewing existing ones before joining as maintainer.